app/experimenter/base/models.py

```diff
@@ -1,6 +1,36 @@
 from django.db import models
 
 
+class SiteFlagNameChoices(models.TextChoices):
+    LAUNCHING_DISABLED = "LAUNCHING_DISABLED", "Disable launching experiments"
+
+
+class SiteFlagManager(models.Manager):
+    def value(self, choice, defval=False):
+        qs = self.get_queryset().filter(name=choice.name)
+        if qs.exists():
+            return qs.get().value
+        return defval
+
+
+class SiteFlag(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    value = models.BooleanField()
+    created_on = models.DateTimeField(auto_now_add=True)
+    modified_on = models.DateTimeField(auto_now=True)
+
+    objects = SiteFlagManager()
+
+    @property
+    def description(self):
+        if self.name in SiteFlagNameChoices:
+            return SiteFlagNameChoices[self.name].label
+        return self.name
```

Comment:
I'd say the description is handy. If only because every time I've dealt with feature flipping shaped things in the past, the IDs easily get obscure and some additional descriptive context is needed eventually.